srcpkgs/linux5.6/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'linux5.6'
 pkgname=linux5.6
-version=5.6.8
+version=5.6.10
 revision=1
 wrksrc="linux-${version}"
 short_desc="Linux kernel and modules (${version%.*} series)"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-only"
 homepage="https://www.kernel.org"
 distfiles="https://cdn.kernel.org/pub/linux/kernel/v5.x/linux-${version}.tar.xz"
-checksum=19e142ff07a2c8d10116074099b9d845682e74544a2b6c1641f7cb044ac282df
+checksum=7bdc96827a9db8de79cd13d74a1c5fe8915674f793e592387f2903cb225801b9
 python_version=2 #unverified
```

Comment:
I believe `python_version=3` is appropriate for `linux5.6` after looking at a few of the scripts installed by `linux5.6-headers`.

Reply:
Thanks for the tip, should the "unverified" comment be removed?

Reply:
Yes. Also, add '[ci skip]' to your commit message to avoid the inevitable Travis timeouts.

Reply:
Thank you